tests/helper/helper_filesystem.go

```diff
@@ -137,11 +137,11 @@ func copyDir(src string, dst string, info os.FileInfo) error {
 		return err
 	}
 
-	return copyFile(src, dst, info)
+	return CopyFile(src, dst, info)
 }
 
-// copyFile copy one file to another location
-func copyFile(src, dst string, info os.FileInfo) error {
+// CopyFile copy one file to another location
+func CopyFile(src, dst string, info os.FileInfo) error {
```

Comment:
This function is being used as a library for function in the same package. So caps in the function name is not required. revert it as it was. For example ```CopyFile``` -> ```copyFile```

Reply:
I have changed it to ```CopyFile``` so that anyone can use this function anywhere as required. Because it is the basic function which can be required in other files.

Reply:
We should not change the working test code until unless there is a hard requirement to do so. In your case you are not using ```CopyFile``` function out side of the same package, then it is not needed at all to open the function for other packages for your changes.